### PR TITLE
Use API minetest for translate

### PR DESCRIPTION
**Pull request by Jat15**

### CODE REVIEW

**Review by lnjX — CHANGES_REQUESTED** [review]
Please remove all old files and update the README.

Also this would prevent us from using weblate. - Why doesn't minetest support po files....
So as long as I can't automatically convert those files and use weblate, I won't merge/use this.

**Comment by Jat15** [comment]
One script for transform po to tr.
And update for 5.0.1

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Minetest Game Internationalization Mod
-MTG 0.5.0-dev (2017-08-28), License: MIT (code), CC 0 (translations)
+MTG 5.0.1 (2017-08-28), License: MIT (code), CC 0 (translations)
```
Comment (lnjX) [inline README.md]: The version number is about the source strings, which haven't been updated.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
 
 ## Description
-A Minetest mod adding translations to the default game of Minetest. You just have to install the
-[intllib](https://github.com/minetest-mods/intllib/) mod and enable both in your world and all item
+A Minetest mod adding translations to the default game of Minetest. You just have enable both in your world and all item
```
Comment (lnjX) [inline README.md]: > You just have enable both in your world and all item

This makes no sense.

Comment (Jat15) [reply README.md]: Propose another sentence.

File: init.lua [hunk 3]
```diff
@@ -1,7 +1,4 @@
--- Load support for intllib.
-local MP = minetest.get_modpath(minetest.get_current_modname())
-local S, NS = dofile(MP.."/intllib.lua")
-
+S = minetest.get_translator("mtg_i18n")
```
Comment (Panquesito7) [inline init.lua]: Why not `local`?
```suggestion
local S = minetest.get_translator("mtg_i18n")
```

File: README.md (hunk 2 above)
Comment (Panquesito7) [reply README.md]: Add a dot at the end of the line.
I think it'll look better.

Comment (Panquesito7) [inline README.md]: Add a dot at the end of the line.
```suggestion
A Minetest mod adding translations to the default game of Minetest. You just have enable both in your world and all item.
```

**Pull request closed without merging**